js/calendar.js

```diff
@@ -854,7 +867,7 @@ if(!String.prototype.formatNum) {
 				return this.locale.title_month.format(this.locale['m' + p.getMonth()], p.getFullYear());
 				break;
 			case 'week':
-				return this.locale.title_week.format(p.getWeek(), p.getFullYear());
+				return this.locale.title_week.format(p.getWeek(getExtentedOption(this, 'week_numbers_iso_8601')), p.getFullYear());
```

Comment:
You add aditional parameter to function but I do not see this function modified to accept it. What is this parameter for?